runtime/mainnet/src/benchmarks.rs

```diff
@@ -42,3 +70,184 @@ frame_benchmarking::define_benchmarks!(
 	[pallet_nfts, Nfts]
 	[pallet_assets, TrustBackedAssets]
 );
+
+parameter_types! {
+	pub ExistentialDepositAsset: Option<Asset> = Some((
+		RelayLocation::get(),
+		ExistentialDeposit::get()
+	).into());
+	pub const AssetHubParaId: ParaId = ParaId::new(1000);
+}
+
+type DeliveryHelper = (
+	cumulus_primitives_utility::ToParentDeliveryHelper<
+		XcmConfig,
+		ExistentialDepositAsset,
+		PriceForParentDelivery,
+	>,
+	polkadot_runtime_common::xcm_sender::ToParachainDeliveryHelper<
+		XcmConfig,
+		ExistentialDepositAsset,
+		PriceForSiblingDelivery,
+		AssetHubParaId,
+		ParachainSystem,
+	>,
+);
+
+impl pallet_xcm::benchmarking::Config for Runtime {
+	type DeliveryHelper = DeliveryHelper;
+
+	fn reachable_dest() -> Option<Location> {
+		Some(Location::parent())
+	}
+
+	fn teleportable_asset_and_dest() -> Option<(Asset, Location)> {
+		// No assets can be teleported.
+		None
+	}
+
+	fn reserve_transferable_asset_and_dest() -> Option<(Asset, Location)> {
+		ParachainSystem::open_outbound_hrmp_channel_for_benchmarks_or_tests(AssetHubParaId::get());
+
+		let who = frame_benchmarking::whitelisted_caller();
+		// Give some multiple of the existential deposit.
+		let balance = ExistentialDeposit::get() * 10_000;
+		let _ =
+			<Balances as frame_support::traits::Currency<_>>::make_free_balance_be(&who, balance);
+		let ah_on_pop: AccountId = LocationToAccountId::convert_location(&AssetHub::get())?;
+		let _ = <Balances as frame_support::traits::Currency<_>>::make_free_balance_be(
+			&ah_on_pop, balance,
+		);
+
+		// We can do reserve transfers of relay native asset to AH.
+		Some((
+			Asset { fun: Fungible(ExistentialDeposit::get()), id: AssetId(Location::from(Parent)) },
```

Comment:
You have a lot of ways you are writing the same thing. Could you please choose a way and be consistent :)

```suggestion
			Asset { fun: Fungible(ExistentialDeposit::get()), id: AssetId(Location::parent()) },
```

Reply:
All locations are now defined in terms of defined types in xcm config file
[dbf83e7](https://github.com/r0gue-io/pop-node/pull/479/commits/dbf83e7915b2a16913e9ea23cf25a324399e5905)